script/gauge.js

```diff
@@ -113,14 +113,21 @@ function _setPerfGaugeExplodey(wrapper, category) {
   let offsetAdder = 0.25 * circumferenceOuter - endDiffOuter - 0.5 * strokeGap;
   let angleAdder = -0.5 * Math.PI;
 
+  // Extra hack on top of the HACK for element reuse below. Delete any metric elems that aren't needed anymore (happens when the same gauge goes from v5 to v6)
+  groupOuter.querySelectorAll('.metric').forEach(metricElem => {
```

Comment:
This is fine but I'll try to follow up with a solution that uses `useEffect` and just clears the `ref` element HTML whenever the version changes.

Reply:
yah seems okay to destroy the gauge elem completely on version change. we just wouldn't want to rebuild its DOM on every metric score change.